### DISCUSSION TITLE
Create first time audio troubleshooting document

### DISCUSSION THREAD

**Discussion by ann0see** [jamulussoftware/jamulus#1405] [Documentation]
We already have a troubleshooting page on the website, but since we're probably moving away from "forcing" people through the documentation and just prominently link a download page, I think having an unofficial troubleshooting PDF you could give your bandmates/choir singers would be great. This should focus on basic problems and guide them through the process. So basically it could be something like a flow chart:

I have sound issues.

Do you hear sound?

Yes:

  Can you be heared by others
    Yes: continue to 2; No: continue to 4

No: 

Check your output settings

etc. 

This would maybe reduce support load during the onboarding process

**Comment by dcorson-ticino-com** [comment]
You mean like this? :
[JamulusQuideEn.pdf](https://github.com/jamulussoftware/jamulus/files/6234999/JamulusQuideEn.pdf)

This is written for the people of our orchestra, but could be easily rewritten for the general case.
(I have that in French and German too)

**Reply by ann0see** [reply]
Kind of, but not exactly. I'll try to make a draft today evening. 

I think it's something like that: https://en.m.wikipedia.org/wiki/Decision_tree

**Comment by ann0see** [comment]
[JamulusErrors.pdf](https://github.com/jamulussoftware/jamulus/files/6238956/JamulusErrors.pdf)

That's a first draft (not yet finished since I'd first like to discuss this idea). We should collect the most common issues and write them down there.

**Reply by gilgongo** [reply]
I take it this is intended to be platform independent? When the general idea of a troubleshooting "wizard" has been discussed before, we've wondered how useful generic help would be without being able to reference, say ASIO or whatever. 

There is also the issue of device dependence in the sense that if I just have an electric quitar plugged in, but no mic, or I do have a mic and that's connected to my right channel, or I have a number of things (mic, guitar, keyboard) but they are plugged into a desk that sends a stereo signal to Jamulus.

Add to this whether we also want to help the user troubleshoot the fact that they may not be observing The Golden Rule?

So it may be more effective if the wizard established these things first so that it could offer more specific help perhaps.

**Comment by tackin** [comment]
Yesterday I discovered that my Zoom H4 pro and latest ASIO-driver does not work on Windows with a Buffer of 64 or 128 Bit. After a long search, I found a confirmation of my problem in a well hidden statement of the manufacturer. It's a known bug for 3 years and no one cares. I doubt, Jamulus can cover all kinds of these stupid things in a troubleshooting "wizard".
Perhaps a "known problems and bugs"-section can cover some of the pitfalls.

**Reply by gilgongo** [reply]
I think you're right. The documentation such as we have it now probably covers the "short head" of problems that perhaps 50% of people who have problems run into, while the long tail of problems the remaining 50% have is very long. We don't have a particularity efficient way of allowing people to check whether their audio device is known to be good though, so perhaps (as your example shows), we might enable that in some way (eg an easily searchable version of [this list](https://jamulus.io/kb/2021/01/05/Jamulus-Sound-Devices.html)).
